.github/workflows/build-and-release-maven.yml

```diff
@@ -0,0 +1,50 @@
+name: Build and Release Maven project
+
+on:
+  workflow_call:
```

Comment:
add each required secret otherwise you are forced to use `secrets: inherit` and actually missing the only way to document each required secret

https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow

Reply:
Done, thanks!